Description: Disable KDE-specific custom-widgets. pykdeuic.py and kde4.py are
 not found, not used and it simply fails.
 - This patch is inherently broken as it forces the use of PyQT4 instead of upstream's use of PyKDE4.
 - This is (IMHO) due to kde-bindings doing something false with python-kde4.
Origin: vendor
Author: Didier Raboud <didier@raboud.com>
Bug: http://bugs.debian.org/540735
Last-Update: 2009-08-10
Index: kdeadmin-4.3.0/system-config-printer-kde/system-config-printer.ui
===================================================================
--- kdeadmin-4.3.0.orig/system-config-printer-kde/system-config-printer.ui	2009-08-10 20:42:41.000000000 +0200
+++ kdeadmin-4.3.0/system-config-printer-kde/system-config-printer.ui	2009-08-10 20:47:43.000000000 +0200
@@ -68,7 +68,7 @@
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
-          <widget class="KPushButton" name="btnNewPrinter">
+          <widget class="QPushButton" name="btnNewPrinter">
            <property name="text">
             <string>New Local Printer</string>
            </property>
@@ -89,7 +89,7 @@
           </widget>
          </item>
          <item row="1" column="0">
-          <widget class="KPushButton" name="btnNewPrinterNetwork">
+          <widget class="QPushButton" name="btnNewPrinterNetwork">
            <property name="text">
             <string>New Network Printer</string>
            </property>
@@ -110,7 +110,7 @@
           </widget>
          </item>
          <item row="2" column="0">
-          <widget class="KPushButton" name="btnNewPrinterSpecial">
+          <widget class="QPushButton" name="btnNewPrinterSpecial">
            <property name="text">
             <string>New Special Printer</string>
            </property>
@@ -131,7 +131,7 @@
           </widget>
          </item>
          <item row="3" column="0">
-          <widget class="KPushButton" name="btnNewClass">
+          <widget class="QPushButton" name="btnNewClass">
            <property name="text">
             <string>New Printer Class</string>
            </property>
@@ -357,7 +357,7 @@
                 </widget>
                </item>
                <item row="0" column="1">
-                <widget class="KLineEdit" name="entPName"/>
+                <widget class="QLineEdit" name="entPName"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="lblPLocation">
@@ -370,7 +370,7 @@
                 </widget>
                </item>
                <item row="1" column="1">
-                <widget class="KLineEdit" name="entPLocation"/>
+                <widget class="QLineEdit" name="entPLocation"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="lblPDescription">
@@ -383,7 +383,7 @@
                 </widget>
                </item>
                <item row="2" column="1">
-                <widget class="KLineEdit" name="entPDescription"/>
+                <widget class="QLineEdit" name="entPDescription"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="lblPDevice2">
@@ -396,7 +396,7 @@
                 </widget>
                </item>
                <item row="3" column="1">
-                <widget class="KLineEdit" name="entPDevice"/>
+                <widget class="QLineEdit" name="entPDevice"/>
                </item>
               </layout>
              </item>
@@ -465,7 +465,7 @@
                 </widget>
                </item>
                <item row="0" column="2">
-                <widget class="KPushButton" name="btnChangePPD">
+                <widget class="QPushButton" name="btnChangePPD">
                  <property name="text">
                   <string>Change...</string>
                  </property>
@@ -570,21 +570,21 @@
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
-                <widget class="KPushButton" name="btnPrintTestPage">
+                <widget class="QPushButton" name="btnPrintTestPage">
                  <property name="text">
                   <string>Print Test Page</string>
                  </property>
                 </widget>
                </item>
                <item>
-                <widget class="KPushButton" name="btnSelfTest">
+                <widget class="QPushButton" name="btnSelfTest">
                  <property name="text">
                   <string>Print Self Test</string>
                  </property>
                 </widget>
                </item>
                <item>
-                <widget class="KPushButton" name="btnCleanHeads">
+                <widget class="QPushButton" name="btnCleanHeads">
                  <property name="text">
                   <string>Clean Print Heads</string>
                  </property>
@@ -608,7 +608,7 @@
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
-                <widget class="KPushButton" name="btnDelete">
+                <widget class="QPushButton" name="btnDelete">
                  <property name="text">
                   <string>Remove Printer</string>
                  </property>
@@ -668,7 +668,7 @@
                 </widget>
                </item>
                <item row="0" column="1">
-                <widget class="KComboBox" name="cmbPStartBanner"/>
+                <widget class="QComboBox" name="cmbPStartBanner"/>
                </item>
                <item row="0" column="2">
                 <spacer name="horizontalSpacer">
@@ -700,7 +700,7 @@
                 </widget>
                </item>
                <item row="1" column="1">
-                <widget class="KComboBox" name="cmbPEndBanner"/>
+                <widget class="QComboBox" name="cmbPEndBanner"/>
                </item>
                <item row="1" column="2">
                 <spacer name="horizontalSpacer_3">
@@ -769,14 +769,14 @@
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
-                <widget class="KPushButton" name="btnPAddUser">
+                <widget class="QPushButton" name="btnPAddUser">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
-                <widget class="KPushButton" name="btnPDelUser">
+                <widget class="QPushButton" name="btnPDelUser">
                  <property name="text">
                   <string>Remove</string>
                  </property>
@@ -1761,23 +1761,6 @@
    </item>
   </layout>
  </widget>
- <customwidgets>
-  <customwidget>
-   <class>KComboBox</class>
-   <extends>QComboBox</extends>
-   <header>kcombobox.h</header>
-  </customwidget>
-  <customwidget>
-   <class>KLineEdit</class>
-   <extends>QLineEdit</extends>
-   <header>klineedit.h</header>
-  </customwidget>
-  <customwidget>
-   <class>KPushButton</class>
-   <extends>QPushButton</extends>
-   <header>kpushbutton.h</header>
-  </customwidget>
- </customwidgets>
  <resources/>
  <connections/>
 </ui>
Index: kdeadmin-4.3.0/system-config-printer-kde/ipp-browse-dialog.ui
===================================================================
--- kdeadmin-4.3.0.orig/system-config-printer-kde/ipp-browse-dialog.ui	2009-08-10 20:48:20.000000000 +0200
+++ kdeadmin-4.3.0/system-config-printer-kde/ipp-browse-dialog.ui	2009-08-10 20:48:29.000000000 +0200
@@ -38,7 +38,7 @@
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
-     <widget class="KPushButton" name="btnIPPBrowseRefresh" >
+     <widget class="QPushButton" name="btnIPPBrowseRefresh" >
       <property name="text" >
        <string>Refresh</string>
       </property>
@@ -58,7 +58,7 @@
      </spacer>
     </item>
     <item>
-     <widget class="KPushButton" name="btnIPPBrowseOk" >
+     <widget class="QPushButton" name="btnIPPBrowseOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
@@ -68,7 +68,7 @@
      </widget>
     </item>
     <item>
-     <widget class="KPushButton" name="btnIPPBrowseCancel" >
+     <widget class="QPushButton" name="btnIPPBrowseCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
@@ -112,13 +112,6 @@
    </column>
   </widget>
  </widget>
- <customwidgets>
-  <customwidget>
-   <class>KPushButton</class>
-   <extends>QPushButton</extends>
-   <header>kpushbutton.h</header>
-  </customwidget>
- </customwidgets>
  <resources/>
  <connections/>
 </ui>
Index: kdeadmin-4.3.0/system-config-printer-kde/smb-browse-dialog.ui
===================================================================
--- kdeadmin-4.3.0.orig/system-config-printer-kde/smb-browse-dialog.ui	2009-08-10 20:47:56.000000000 +0200
+++ kdeadmin-4.3.0/system-config-printer-kde/smb-browse-dialog.ui	2009-08-10 20:48:14.000000000 +0200
@@ -38,7 +38,7 @@
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
-     <widget class="KPushButton" name="btnSMBBrowseRefresh" >
+     <widget class="QPushButton" name="btnSMBBrowseRefresh" >
       <property name="text" >
        <string>Refresh</string>
       </property>
@@ -58,7 +58,7 @@
      </spacer>
     </item>
     <item>
-     <widget class="KPushButton" name="btnSMBBrowseOk" >
+     <widget class="QPushButton" name="btnSMBBrowseOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
@@ -68,7 +68,7 @@
      </widget>
     </item>
     <item>
-     <widget class="KPushButton" name="btnSMBBrowseCancel" >
+     <widget class="QPushButton" name="btnSMBBrowseCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
@@ -112,13 +112,6 @@
    </column>
   </widget>
  </widget>
- <customwidgets>
-  <customwidget>
-   <class>KPushButton</class>
-   <extends>QPushButton</extends>
-   <header>kpushbutton.h</header>
-  </customwidget>
- </customwidgets>
  <resources/>
  <connections/>
 </ui>
